src/Loader/AbstractFileLoader.php

```diff
@@ -57,20 +60,19 @@ protected function findRelativePath(string $path)
      */
     protected function findAbsolutePath(string $path)
     {
-        if (is_file($path) || is_dir($path)) {
+        if (file_exists($path)) {
             return realpath($path);
         }
 
         if ($this->basePath === null) {
             return false;
         }
 
-        if (is_file($this->basePath . DIRECTORY_SEPARATOR . $path)
-               || is_dir($this->basePath . DIRECTORY_SEPARATOR . $path)) {
-            return realpath($this->basePath . DIRECTORY_SEPARATOR . $path);
+        try {
+            return Filesystem::getRealPath($this->basePath . DIRECTORY_SEPARATOR . $path);
+        } catch (FilesystemException) {
+            return false;
```

Comment:
Sorry if I wasn't clear before, but I think this usage should probably also just stay as realpath, since we don't gain anything by moving to the new helper here.